docs/05-the-editor/10-shortcuts/index.md

```diff
@@ -28,6 +28,7 @@ When an element or component is selected in the canvas or element tree:
 
 - Hold [kbd]Alt[kbd] while dropping a dragged element in the element tree - Copy the element instead of moving it
 - Hold [kbd]Alt[kbd] while dragging an element in the canvas - Drag a duplicate of the selected element/component instead of moving the original
```

Comment:
```suggestion
- Hold [kbd]Option/Alt[kbd] while dropping a dragged element in the element tree - Copy the element instead of moving it
- Hold [kbd]Option/Alt[kbd] while dragging an element in the canvas - Drag a duplicate of the selected element/component instead of moving the original
```